src/main.cpp

```diff
@@ -2,12 +2,13 @@
 #include <Arduino.h>
 #include <BleMouse.h>
 #include <BLEDevice.h>
-#include "usb_host.h"
+#include "usb_hid_host.h"
```

Comment:
How to use the keyboard?
BleMouse.h & BleKeyboard.h CANNOT be used together.
If we want both (or even add joystick :-) ), we need another BLE mouse,
maybe ours again (esp32_mouse_keyboard).